vhdbuilder/packer/vhd-image-builder-base.json

```diff
@@ -56,6 +56,8 @@
       "location": "{{user `location`}}",
       "vm_size": "{{user `vm_size`}}",
       "polling_duration_timeout": "1h",
+      "managed_image_name": "{{user `sig_image_name`}}-{{user `captured_sig_version`}}",
```

Comment:
did we want to add this back too? `"managed_image_storage_account_type": "Premium_LRS",`?

Reply:
I was just going to wait on that a bit because Im seeing its effect with the pulling container images and bcc installation time regression.

Hoping to merge my benchmarks PR then work on that.